client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -854,6 +854,17 @@ Undocumented.prototype.getStoredCards = function( fn ) {
 	return this.wpcom.req.get( { path: '/me/stored-cards' }, fn );
 };
 
+/**
+ * Get a list of the user's stored payment methods
+ *
+ * @param {object} query The query parameters
+ * @param {Function} fn The callback function
+ */
```

Comment:
Might be worth mentioning in the docblock that this will return a Promise if `fn` is not present (a slightly hidden feature of `wpcom.req` functions that is good to know about!).